public/js/components/submit-button.jsx

```diff
@@ -6,16 +6,21 @@ export default class SubmitButton extends Component {
 
   static propTypes = {
     isDisabled: PropTypes.bool,
+    modifier: PropTypes.string.isRequired,
     showSpinner: PropTypes.bool,
     text: PropTypes.string.isRequired,
   }
 
+  static defaultProps = {
+    modifier: null,
+  }
+
   render() {
     var { isDisabled, text, showSpinner, ...buttonAttrs } = this.props;
 
     var buttonClassNames = cx({
       'spinner': showSpinner,
-    });
+    }, this.props.modifier);
```

Comment:
Changed to cssModifier (modifier is a commonly used term for css classes added to modify the look of something).